### PR TITLE
refactor: use fontSize array for font sizes within app

### LINKED ISSUE #6: Create fontSize theme export so we don't use padding to denote text size
Padding isn't fontSize, so we need to create an export of several sizes to reign in the rampant use of padding values.

### PR DESCRIPTION
**Pull request by Danondso**
Closes #6

### CODE REVIEW

File: src/screens/Details/Details.styles.ts [hunk 1]
```diff
@@ -40,20 +40,20 @@ export default StyleSheet.create<Style>({
   },
   artistName: {
     padding: padding.eighth,
-    fontSize: padding.base,
+    fontSize: fontSizes[0],
```
Comment (jaredswinney) [inline src/screens/Details/Details.styles.ts:43]: Magic indexes lead to unreadable code. Could this be a map with ENUM keys like "SMALL" "MEDIUM" "LARGE" "X-LARGE"

This way if you wanted to add an X-SMALL (something smaller than fontSizes[0], you wouldn't have to reindex all references as +1

File: src/theme/index.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import defaultPalette from './styles';
 
-const BASE: number = 16;
+const fontSizes = [16, 24, 32, 48];
 
-// TODO, make a fontSize equivalent here
+const BASE: number = 16;
```
Comment (jaredswinney) [inline src/theme/index.ts:5]: BASE what?

File: src/theme/index.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import defaultPalette from './styles';
 
-const BASE: number = 16;
+const fontSizes = [16, 24, 32, 48];
```
Comment (jaredswinney) [inline src/theme/index.ts:3]: inconsistent const naming conventions. Typically consts/statics are UPPER_SNAKE_CASE, but at least have them match imo

**Review by jaredswinney — CHANGES_REQUESTED** [review]

File: src/screens/Details/Details.styles.ts (hunk 1 above)
Comment (Danondso) [reply src/screens/Details/Details.styles.ts:43]: I like that, tbh I haven't used TS enums yet but this defs fits that use case.

File: src/theme/index.ts (hunk 2 above)
Comment (Danondso) [reply src/theme/index.ts:5]: PADDING sir, PADDING

File: src/theme/index.ts (hunk 3 above)
Comment (Danondso) [reply src/theme/index.ts:3]: Updated all of them in the theme file.

File: src/theme/index.ts [hunk 4]
```diff
@@ -1,30 +1,35 @@
 import defaultPalette from './styles';
 
-const BASE: number = 16;
+enum FONT_SIZES {
+  SMALL = 16,
```
Comment (jaredswinney) [inline src/theme/index.ts:4]: I forgot that js works this way. the Java has infected me

**Review by jaredswinney — APPROVED** [review]

**Pull request merged**